_delphi_utils_python/tests/test_geomap.py

```diff
@@ -104,6 +104,14 @@ class TestGeoMapper:
             ),
         )
     )
+    mega_data_3 = pd.DataFrame(
+        {
+            "fips": [1123, 1125, 1126, 1128, 1129, 18181],
+            "timestamp": [pd.Timestamp("2018-01-01")] * 6,
+            "count": [2, 1, 5, 7, 3, 10021],
+            "visits": [4, 1, 2, 5, 10, 100001],
```

Comment:
making this easier to interpret in the diff
```suggestion
            "visits": [4, 1, 2, 5, 10, 100001],
            "count": [2, 1, 5, 7, 3, 10021],
```